docs/multicluster/user-guide.md

```diff
@@ -349,8 +355,18 @@ the following command to annotate Node `node-1` as the Multi-cluster Gateway:
 $kubectl annotate node node-1 multicluster.antrea.io/gateway=true
 ```
 
-Multi-cluster Controller in the member cluster will detect the Gateway Node, and
-create a `Gateway` CR with the same name as the Node. You can check it with command:
+You can annotate multiple Nodes in a member cluster as the candidates for
+Multi-cluster Gateway, but only one Node will be selected as the active Gateway.
+Before Antrea v1.9.0, the Gateway Node is just randomly selected and will never
+change unless the Node or its `gateway` annotation is deleted. Starting with
```

Comment:
Starting from?

Reply:
I think both are ok, but "starting with" is more popular. Also most our docs use "starting with".

Reply:
Got it, thanks.